.github/workflows/docker.yml

```diff
@@ -0,0 +1,71 @@
+
+name: Docker CI/CD
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+  pull_request:
```

Comment:
should we run on each commit of pull request?

Reply:
@shan-96 
- I think for pull request we should build, but not push, to be sure that we're not merging failing build
- on every comment to master we should build and push with tag latest
- on release/tag we should build and push with tag name, that will contain version number.

make sense?

Reply:
lets follow this style 
![image](https://github.com/user-attachments/assets/9dcc5574-db5a-481a-9246-edbb5247dc6a)

Reply:
looks good